sci-electronics/kicad/kicad-4.0.0_rc1.ebuild

```diff
@@ -0,0 +1,178 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
+PYTHON_COMPAT=( python2_7 )
+
+WX_GTK_VER="3.0"
+
+inherit bzr cmake-utils fdo-mime flag-o-matic gnome2-utils python-r1 vcs-snapshot wxwidgets versionator
```

Comment:
what exactly is the bzr eclass used for? Both it's exported functions are overwritten and I don't see that we use any of the variables either. In fact, we even get an unconditional build-time dep on bzr due to this, which even seems incorrect.